src/client/pages/Landing/components/Card.tsx

```diff
@@ -128,11 +132,58 @@ const CardContainer: React.FC<{
   )
 }
 
-export const Card: React.FC<{
+export const CardHeadline = styled.h2<{ disabled?: boolean }>`
+  width: 100%;
+  margin: 0;
+  font-size: 1.25rem;
+  line-height: 1.2;
+  color: ${(props) => (props.disabled ? colorsV3.gray500 : colorsV3.gray900)};
+
+  @media (min-width: 600px) {
+    margin-bottom: 0.25rem;
+    font-size: 1.5rem;
+    line-height: 1.25;
+  }
+
+  @media (min-width: 1020px) {
+    font-size: 1.9rem;
```

Comment:
This solves the layout issue for the DK case, just wondering if we should go outside our defined font-size? (You don't know about them of course so hard for you to know:D) https://www.figma.com/file/zlX4FcM5dMRz9stsCNQJXB/Web-UI-Kit?node-id=0%3A1 I created a review app so we can let Mange have a look.